esp/templates/program/modules/volunteermanage/main.html

```diff
@@ -29,8 +29,9 @@
     <tr>
     <td>{{ vr.timeslot.pretty_time }}</td>
     <td>{{ vr.timeslot.description }}: {{ vr.num_offers }} / {{ vr.num_volunteers }} volunteers <br />
-        <div id="offers" style="color: blue; font-size: 0.9em;">{% for offer in vr.get_offers %}
-        {{ offer.name }}, {{ offer.email }}, {{ offer.phone }} <br />
+        <div id="offers" style="color: blue; font-size: 0.9em;">
+        {% for offer in vr.get_offers %}
+        <a href="{{ offer.user.get_absolute_url }}">{{ offer.name }}</a>, {{ offer.email }}, {{ offer.phone }} <br />
```

Comment:
Nope!  If we're not requiring login, we just take their name and email and leave it at that.